### PR TITLE
M3-5675: Post-demo Refinements to Access Controls

### PR DESCRIPTION
**Pull request by DevDW**
## Description
- [x] Revisions to copy and drawer & button names
- [x] Accept and enforce CIDR notation in drawer (same convention as Firewalls)
- [x] Loading state on "Remove IP Address" in Remove IP confirmation modal

## How to test
- When you go to remove an IP address, confirm there is a button loading state, and that errors display as expected
- Check that copy matches what is specified in ticket
- Confirm that CIDR notation (`/32`) is accepted and enforced on IPs in the drawer

### CODE REVIEW

File: packages/manager/src/features/Databases/DatabaseDetail/AccessControls.tsx [hunk 1]
```diff
@@ -90,6 +90,9 @@ export const AccessControls: React.FC<Props> = (props) => {
 
   const [isDialogOpen, setDialogOpen] = React.useState<boolean>(false);
   const [error, setError] = React.useState<string | undefined>();
+  const [ipRemovalLoading, setIPRemovalLoading] = React.useState<boolean>(
+    false
+  );
```
Comment (HanaXu) [inline packages/manager/src/features/Databases/DatabaseDetail/AccessControls.tsx:95]: We can use the loading state from `useDatabaseMutation` instead

```typescript
const {
    isLoading: databaseUpdating,
    mutateAsync: updateDatabase,
  } = useDatabaseMutation(engine, id);
```

File: packages/manager/src/features/Databases/DatabaseDetail/AddAccessControlDrawer.tsx [hunk 2]
```diff
@@ -123,26 +141,44 @@ const AddAccessControlDrawer: React.FC<CombinedProps> = (props) => {
...
+              />
+            ))
+          : null}
         <Typography variant="body1" className={classes.instructions}>
-          To restrict connections to trusted sources, add at least one inbound
-          source below.{' '}
-          <ExternalLink to="https://www.linode.com/docs/products/database">
+          Add or remove IPv4 addresses and ranges that should be authorized to
+          view your cluster&apos;s database.{' '}
+          <ExternalLink to="https://www.linode.com/docs/products/databases/managed-databases/">
```
Comment (HanaXu) [inline packages/manager/src/features/Databases/DatabaseDetail/AddAccessControlDrawer.tsx:168]: Would the link https://www.linode.com/docs/products/databases/managed-databases/guides/manage-access-controls/ make more sense here? For reference, see MWildman's comment in M3-5685

File: packages/manager/src/features/Databases/DatabaseDetail/AccessControls.tsx [hunk 3]
```diff
@@ -190,7 +201,7 @@ export const AccessControls: React.FC<Props> = (props) => {
               Add the IP addresses for other instances or users that should have
               the authorization to view this cluster&apos;s database. By
               default, all public and private connections are denied.{' '}
-              <ExternalLink to="https://www.linode.com/docs/products/database">
+              <ExternalLink to="https://www.linode.com/docs/products/databases/managed-databases/">
```
Comment (HanaXu) [inline packages/manager/src/features/Databases/DatabaseDetail/AccessControls.tsx:204]: Seems a bit odd to link to the general product page here 🤔

File: packages/manager/src/features/Databases/DatabaseDetail/AddAccessControlDrawer.tsx (hunk 2 above)
Comment (DevDW) [reply packages/manager/src/features/Databases/DatabaseDetail/AddAccessControlDrawer.tsx:168]: Yes, I'd say that's more appropriate here. Wasn't aware of that link. Thanks!

**Review by bnussman — APPROVED** [review]

**Review by HanaXu — APPROVED** [review]

**Pull request merged**